assets/javascripts/discourse/models/user-theme-settings.js

```diff
@@ -0,0 +1,13 @@
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import ThemeSettings from "admin/models/theme-settings";
+
+export default class UserThemeSettings extends ThemeSettings {
+  loadMetadata(themeId) {
+    return ajax(
+      `/user_themes/${themeId}/objects_setting_metadata/${this.setting}.json`
+    )
+      .then((result) => this.set("metadata", result))
+      .catch(popupAjaxError);
+  }
```

Comment:
we need to make the metadata accessible to theme owners even if they are not admins